eosiojavasoftkeysignatureprovider/src/main/java/one/block/eosiosoftkeysignatureprovider/SoftKeySignatureProviderImpl.java

```diff
@@ -69,6 +62,12 @@
      */
     private static final int BIG_INTEGER_POSITIVE = 1;
 
+    /**
+     * Place holder for Whether getAvailableKeys should return WIF legacy format for key generated with secp256k1
+     * algorithm.  This for of the key is prefaced with "EOS" instead of "PUB_K1_".
```

Comment:
Change this to read:    "Flag to indicate whether getAvailableKeys() should return keys generated with the secp256k1 algorithm in the legacy (prefaced with "EOS") or new (prefaced with "PUB_K1_") formats."